frontend/src/secure_document_upload/components/CreditTransferRequestTable.js

```diff
@@ -39,9 +39,19 @@ const CreditTransferRequestTable = (props) => {
     id: 'status',
     minWidth: 50
   }, {
-    className: 'col-category',
-    Header: 'Category',
-    id: 'category',
+    accessor: item => {
+      if (item.type.theType === 'Evidence') {
+        if (item.milestone !== null && item.milestone.milestone.length > 0) {
```

Comment:
I seem to be getting an error with the check if the document evidence is missing a milestone:
```Cannot read property 'length' of null```

The check might need to be changed to:
```if (item.milestone && item.milestone.milestone) {```
or just
```if (item.milestone) {```

Reply:
@amichard Fixed with the latest push